install/installer/README.md

```diff
@@ -163,12 +163,21 @@ yq eval-all --inplace \
   gitpod.yaml
 ```
 
+Similarly, if you are doing a `Workspace` only install (specifying `Workspace`
+as the kind in config) you might want to change the service type of `ws-proxy` to `ClusterIP`
+instead of the default `LoadBalancer`. You can post-process the YAML to change that.
+
+```shell
+yq eval-all --inplace \
+  '(select(.kind == "Service" and .metadata.name == "ws-proxy") | .spec.type) |= "ClusterIP"' \
+  gitpod.yaml
+```
```

Comment:
Off-topic: Do we have already an issue to get rid of this post-processing step and having a config value for the service type?

Reply:
@corneliusludmann there is no such issue open specifically for making the service type configurable. I guess the question is do we plan on making this configurable, since our thought process [was to not make it configurable.](https://github.com/gitpod-io/gitpod/pull/7280)

Reply:
I'm glad you brought that up. :rocket: Indeed, we decided like this in the past. However, 2 things have changed since then:
- We introduced the `experimental` section. Everything that we won't support for some reason but need internally for SaaS should go there. We shouldn't need to rely on post-processing.
- Since our official supported installation path will be the KOTS installation, we are a little more relaxed about the config surface.

What do you think, @MrSimonEmms?

Reply:
Sorry for the inactivity on this issue. I have created[ this issue ](https://github.com/gitpod-io/gitpod/issues/8838) to track this requirement. Shall we merge this PR and continue the work there?